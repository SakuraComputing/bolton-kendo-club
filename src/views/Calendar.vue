<template>
    <div class="calendar">
        <h1>Kendo Calendar</h1>
    </div>    
</template>

<style>

.kendo-video {
    margin: 8%;
}
</style>

