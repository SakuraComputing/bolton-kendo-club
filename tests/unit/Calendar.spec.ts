import { shallowMount } from '@vue/test-utils'
import Calendar from '@/views/Calendar.vue'

describe('Calendar.vue', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(Calendar);
    expect(wrapper).toMatchSnapshot()
  })
})
