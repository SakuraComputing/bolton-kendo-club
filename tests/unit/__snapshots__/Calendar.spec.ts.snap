// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calendar.vue renders correctly 1`] = `
<div class="calendar">
  <h1>Kendo Calendar</h1>
</div>
`;
